app/services/sso/openid_connect.rb

```diff
@@ -0,0 +1,61 @@
+module SSO
+  class OpenidConnect < Base
+    delegate :session, :to => :controller
+    attr_reader :current_user
+
+    def available?
+      controller.api_request? && bearer_token_set? && valid_issuer?
+    end
+
+    def authenticate!
+      payload = jwt_token.decode
+      user = find_or_create_user_from_jwt(payload)
+      @current_user = user
+      update_session(payload)
+      user&.login
+    rescue JWT::DecodeError
+      Rails.logger.error "JWT SSO: Failed to decode JWT."
+      nil
```

Comment:
Is this actually hiding the fact that authentication is failing on ruby 2.3? 
I would expect the test to fail on 2.3 since it can't decode the token, so either there is an issue in the test or we're somehow hiding the fact that it fails?